src/cpulists/parse.go

```diff
@@ -203,3 +204,44 @@ func handleSingleCPU(item string, cpus CPUs, t int) error {
 	cpus[cpu] = true
 	return nil
 }
+
+func GenCPUlist(cpus map[int]bool) string {
```

Comment:
I don't see any benefit of using a map here. It has just added complexity on both producer and consumer sides.
```suggestion
func GenCPUlist(cpus []int) string {
```

Reply:
The map, it's being used across this module all over the place. It's easier to translate from and understand it's meaning (the key indicates the CPU, the non-existence of the key means that the CPU isn't there).

The use of a slice of integers would have to pass to a translation, and it's confuse. If I change here, I would have to change the way that the entire module works.